e2e/support/gomega_support.go

```diff
@@ -0,0 +1,57 @@
+//go:build integration
+// +build integration
+
+// To enable compilation of this file in Goland, go to "Settings -> Go -> Vendoring & Build Tags -> Custom Tags" and add "integration"
+
+/*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package support
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	. "github.com/onsi/gomega"
+)
+
+const (
+	eventuallyTimeoutEnvVarName           = "E2E_DEFAULT_EVENTUALLY_TIMEOUT"
+	eventuallyPollingIntervalEnvVarName   = "E2E_DEFAULT_EVENTUALLY_POLLING_INTERVAL"
+	consistentlyDurationEnvVarName        = "E2E_DEFAULT_CONSISTENTLY_DURATION"
+	consistentlyPollingIntervalEnvVarName = "E2E_DEFAULT_CONSISTENTLY_POLLING_INTERVAL"
```

Comment:
Here I have introduced some additional env vars to that mirrors the `GOMEGA_` ones, not sure if is the best choice or if it would have been better to re-use the same env vars